app/controllers/workflows_controller.rb

```diff
@@ -66,6 +67,10 @@ def destroy
 
   private
 
+  def context
+    params[:context] || nil
```

Comment:
I guess it could, but reading the body this way gives you a string.  `params[:context]` gives you the hash you want to work with(as `ActionController::Parameters`, nicely encoded in the `context` block of the params, so they don't conflict with any other params on the form).

Reply:
I'm thinking the whole body is the context without having an explicit `context` key.  So alternatively, `params.except(:version, :druid)`.